JASP-Desktop/widgets/listmodelmultitermsassigned.cpp

```diff
@@ -27,6 +27,7 @@ ListModelMultiTermsAssigned::ListModelMultiTermsAssigned(QMLListView* listView,
 	, _columns(columns)
 {
 	_copyTermsWhenDropped = true;
+	_allowDuplicatesInMultipleColumns = listView->getItemProperty("allowDuplicatesInMultipleColumns").toBool();
```

Comment:
So the value will not be updated afterwards if it changes right?
Maybe we should put this kind of info next to the properties in qml. with a comment or something.

Reply:
Hey Joris, you are still working?
I have added a comment